web/sites/default/settings.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+$settings['hash_salt'] = "testsalt";
```

Comment:
Nope! We can't put the hash into the settings.php. That should be set in your settings.local.php only.